functions/register.go

```diff
@@ -68,3 +72,52 @@ func Register(token string) error {
 	fmt.Printf("registered with server %s\n", serverData.Server)
 	return nil
 }
+
+func doubleCheck(host *config.Config, apiServer string) (shouldUpdate bool, err error) {
```

Comment:
This function is not required.  CheckConfig is callled at start of every netclient command; if there is a check here that is not in that function, it should be added to that function

Reply:
this was added as duct tape a while ago, somehow some of these host fields were not populated. maybe you can fix checkconfig in future, leaving for now.